sha256-p+f670JPy1+D2Prs+daXozjaf3qQb8GvvA4Yee8xvVM=--- MODULE.bazel
+++ MODULE.bazel
@@ -0,0 +1,5 @@
+module(
+    name = "glfw",
+    version = "3.3.9",
+    compatibility_level = 0,
+)
